js/phobos.js

```diff
@@ -13,6 +13,48 @@ function lastName(a) {
     return a.slice(pos + 1);
 }
 
+function showTooltip(elem, msg){
+    elem.setAttribute('class','btn tooltipped tooltipped-s');
+    elem.setAttribute('aria-label',msg);
+    elem.addEventListener('mouseleave',function(e){
+        //e.currentTarget.setAttribute('class','btn');
+        e.currentTarget.removeAttribute('aria-label');
+    });
+}
+
+function createClipboard(a, symbolName)
+{
+    // copy-to-clipboard
+    var el = document.createElement("i");
+    // name start with .
+    var symbol = a.name.split(".")[1];
+    var text = "import " + document.body.id + " : " + symbol + ";";
+    el.setAttribute("data-clipboard-text", text);
+    el.setAttribute("aria-hidden", true);
+    el.className = "fa fa-clipboard";
+
+    var container =  document.createElement("span");
+    container.className = "tip smallprint decl_relement";
+    container.setAttribute("width", "auto");
```

Comment:
This was only a test - as you can [see](http://dtest.thecybershadow.net/artifact/website-94acb02e4bb4c63b7b53c1fa2a531d29496be7ae-77a9a64d4957722ede41343ea2fdc28d/web/phobos-prerelease/std_algorithm_searching.html) we still need to fix the width & alignment of the tooltip.